packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,281 @@
+import * as React from 'react';
+import { ActionButton, IButtonStyles } from 'office-ui-fabric-react/lib/Button';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { ApiReferencesTable, MEDIUM_GAP_SIZE, LARGE_GAP_SIZE } from './ApiReferencesTable';
+import { IApiProperty, IApiInterfaceProperty, IApiEnumProperty, IMethod, IApiReferencesTableSetProps } from './ApiReferencesTableSet.types';
+import { IEnumTableRowJson, ITableRowJson, ITableJson } from 'office-ui-fabric-react/lib/common/DocPage.types';
+import { PropertyType } from '../../utilities/parser/index';
+import { extractAnchorLink } from '../../utilities/extractAnchorLink';
+
+export interface IApiReferencesTableSetState {
+  properties: Array<IApiProperty>;
```

Comment:
These should be in a private member, not the state. State is only for things that can change during the life of the component (and ideally only changes from things like user interaction or server data fetching, not props updates).